tests/test_crawlers.py

```diff
@@ -312,6 +312,38 @@ def test_intersects_time_range_no_upper_limit(self):
         self.assertFalse(crawler._intersects_time_range(None, datetime(2019, 2, 19)))
 
 
+class LocalDirectoryCrawlerTestCase(unittest.TestCase):
+    """Tests for LocalDirectoryCrawler"""
+
+    def setUp(self):
+        self.crawler = crawlers.LocalDirectoryCrawler('')
+
+    def test_list_folder_contents(self):
+        """_list_folder_contents() should return the absolute
+        path of all files contained in the folder"""
+        with mock.patch('os.listdir', return_value=['foo', 'bar', 'baz']):
+            self.assertListEqual(
+                self.crawler._list_folder_contents('/tmp'),
+                ['/tmp/foo', '/tmp/bar', '/tmp/baz',]
```

Comment:
this test will not work on the windows machine! can you fix it by os.join?

Reply:
I will have a look